firebase-firestore/src/main/java/com/google/firebase/firestore/PersistentCacheIndexManager.java

```diff
@@ -52,4 +52,12 @@ public void enableIndexAutoCreation() {
   public void disableIndexAutoCreation() {
     client.disableIndexAutoCreation();
   }
+
+  /**
+   * Removes all persistent cache indexes. Please note this function will also deletes indexes
+   * generated by {@link FirebaseFirestore#setIndexConfiguration(String)}.
```

Comment:
We are removing this method, so no need to mention here.

Reply:
however when the feature releases, the deprecated setIndexConfiguration is still available to use. I would like to add notes mentioning the function is deprecated. Please let me know if you feel strong I should remove this comment.